ihatemoney/tests/tests.py

```diff
@@ -298,6 +301,12 @@ def test_membership(self):
         result = self.client.get("/raclette/")
         self.assertIn("fred", result.data.decode('utf-8'))
 
+        # Empty bill list should now propose to add bills
+        self.assertIn(
+            'You probably want to <a href="/raclette/add"',
+            result.data.decode('utf-8')
```

Comment:
For this test it seems to be an acceptable place, but for the other, I would prefer to put it somewhere else :-)

Reply:
move them to a specific test function.
tell me if it's what you meant ;-)